python/openapi.json

```diff
@@ -903,6 +907,56 @@
         ]
       }
     },
+    "/api/ai/{project_id}/chat_stream": {
+      "post": {
+        "operationId": "http_ai_chat_stream_api_ai__project_id__chat_stream_post",
+        "parameters": [
+          {
+            "in": "path",
+            "name": "project_id",
+            "required": true,
+            "schema": {
+              "title": "Project Id",
+              "type": "string"
+            }
+          }
+        ],
+        "requestBody": {
+          "content": {
+            "application/json": {
+              "schema": {
+                "$ref": "#/components/schemas/ChatRequest"
+              }
+            }
+          },
+          "required": true
+        },
+        "responses": {
+          "200": {
+            "content": {
+              "application/json": {
+                "schema": {}
```

Comment:
This isn't correct. It's another "incompatibility" with the codegen and fast api internal types (here StreamingResponse).

Reply:
To clarify, we can merge this. Calling this incorrect is a bit to much. I just wanted to note that we should try to have a way to type this response, but I understand how hard it can be ([swagger spec](https://github.com/OAI/OpenAPI-Specification/issues/1576)).

Reply:
I'm happy to add a one-off line in our schema generation code to handle this endpoint (and file upload) if you feel there's a correct way to handle it.

Reply:
Can we configure the stream method with the following:

```
@router.post(
    "/{project_id}/chat_stream",
    responses={
        200: {
            "content": { "text/event-stream": {} },
            "description": "Stream chat reply.",
        }
    },
)
```

This would make this information available here to be used in a follow-up issue to improve the "auto-api" in the client for text stream API calls. CC @danvk